### DISCUSSION TITLE
just decode a QR code from a camera stream and consol a inforamtion

### DISCUSSION THREAD

**Discussion by NileshSapkale301** [zxing-cpp/zxing-cpp#873] [Q&A]
can you please tell me how i can do this.
i want to read a qr code and print a information. how i can do.
i just a want to use a c program i don't want to use any library.

**Comment by axxel** [comment]
Simply use the `ZXingReader` from the examples folder. If it needs to be a c program, you can have a look [here](https://github.com/zxing-cpp/zxing-cpp/blob/master/wrappers/c/ZXingCTest.c). Both use the `zxing-cpp` library of course. But you can build it statically linked, if that is what you mean by "not using any library".
